Automattic/jetpack
# Sharing: remove Reddit's official sharing button

Fixes #24733

## Proposed changes:

The iFramed button hasn't worked for more than a year. Let's remove it and use the default icon+text sharing button instead, this one still works.

If Reddit ever fixes that iFramed button, we can add it back.

For reference, we contacted Reddit about the problem here:
[https://www.reddit.com/r/redditdev/comments/vc6x31/reddit_sharing_button_updates/](https://www.reddit.com/r/redditdev/comments/vc6x31/reddit_sharing_button_updates/)

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* Since we're removing the iFrame, this removes any data that Reddit may track about the button on their end.

## Testing instructions:

* Go to Jetpack > Settings > Sharing, and enable sharing buttons
* Go to Settings > Sharing
* Add a Reddit sharing button
* Pick the official sharing button style.
    * The Reddit button should look good in the preview.
    * It should look good on the frontend as well.
    * When clicking, sharing to Reddit should still work.
